### PR TITLE
Crash when opening Contacts screen

### LINKED ISSUE #910: Crash when opening Contacts screen
App crashes every time when I try to open Contacts screen. It's `Realm accessed from incorrect thread` error again

Screenshot 2021-11-04 at 11 42 44

### PR DESCRIPTION
**Pull request by ivan-ushakov**
close #910

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Review by tomholub — DISMISSED** [review]

**Comment by tomholub** [comment]
for reference - I'm disabling drafts #975 which will fix this issue for now. That means no action needed in ui tests regarding drafts for now, hopefully.

**Comment by tomholub** [comment]
drafts disabled on master, let's see

**Comment by tomholub** [comment]
I've removed the drafts pass phrase issue. @ivan-ushakov please see tomorrow about running that test. You can run just that one test with 

```
npm run-script only.test.filter "user is able to select recipient"
```

**Comment by sosnovsky** [comment]
I noticed new Realm feature for thread safety - https://docs.mongodb.com/realm/sdk/ios/advanced-guides/threading/#use-the--threadsafe-wrapper, may be it'll be useful for such cases with `Realm accessed from incorrect thread` errors?

**Comment by ivan-ushakov** [comment]
> I noticed new Realm feature for thread safety - https://docs.mongodb.com/realm/sdk/ios/advanced-guides/threading/#use-the--threadsafe-wrapper, may be it'll be useful for such cases with `Realm accessed from incorrect thread` errors?

This could help but I think that we need to separate data layer and business logic layer anyway

File: appium/tests/specs/composeEmail/SelectRecipientByName.spec.ts [hunk 1]
```diff
@@ -15,12 +15,18 @@ describe('COMPOSE EMAIL: ', () => {
 
     it('user is able to select recipient from contact list using contact name', () => {
 
-        const contactEmail = CommonData.contact.email;
-        const contactName = CommonData.contact.name;
+        const firstContactEmail = CommonData.contact.email;
+        const firstContactName = CommonData.contact.name;
+        const firstContactItemName = 'Dmitry at FlowCrypt';
```
Comment (ivan-ushakov) [inline appium/tests/specs/composeEmail/SelectRecipientByName.spec.ts:20]: For some reason contact name and Contacts screen item name are different values

Comment (tomholub) [reply appium/tests/specs/composeEmail/SelectRecipientByName.spec.ts:20]: That may be possible. Eg, maybe the contact search name comes from Google Contacts but this name comes from PGP key user id.

File: appium/tests/specs/composeEmail/SelectRecipientByName.spec.ts [hunk 2]
```diff
@@ -35,12 +41,21 @@ describe('COMPOSE EMAIL: ', () => {
...
         InboxScreen.clickCreateEmail();
 
-        NewMessageScreen.setAddRecipientByName(contactName, contactEmail);
-        NewMessageScreen.checkAddedRecipient(contactEmail);
+        NewMessageScreen.setAddRecipientByName(firstContactName, firstContactEmail);
+        NewMessageScreen.checkAddedRecipient(firstContactEmail);
         NewMessageScreen.clickBackButton();
 
+        // Add second contact
+        InboxScreen.clickCreateEmail();
```
Comment (tomholub) [inline appium/tests/specs/composeEmail/SelectRecipientByName.spec.ts:52]: Could have added a second recipient on the same screen. It's a minor detail, but would speed up the test a little bit.

**Review by tomholub — APPROVED** [review]
Thank you for adjusting the test! As this codebase matures, it will be increasingly important to maintain our ui tests, learn to write and debug and update them, for every developer. Very happy to see the this becoming a reality.

**Pull request merged**